src/addresses/arbitrum/connectorsV2_M1.ts

```diff
@@ -0,0 +1,6 @@
+export const connectorsV2_M1 = {
+    "AUTHORITY-A": "0x6ce3e607c808b4f4c26b7f6adaeb619e49cabb25",
+    "BASIC-A": "0x9926955e0dd681dc303370c52f4ad0a4dd061687",
+    "BASIC-B": "0xa9b99766e6c676cf1975c0d3166f96c0848ff5ad",
+    "BASIC-C": "0x839c2d3ade63df5b0b8f3e57d5e145057ab41556"
```

Comment:
```suggestion
    "UNISWAP-V3-A": "0x3254Ce8f5b1c82431B8f21Df01918342215825C2"
```
@ishanjain28 https://github.com/Instadapp/dsa-connectors/pull/82#issuecomment-923794557